### DISCUSSION TITLE
2023/04/03/%E9%94%90%E8%AF%84/%E5%AE%B6%E6%97%8F%E8%AE%A1%E7%94%BB/

### DISCUSSION THREAD

**Discussion by giscus** [dangokanako/blog-comments-gitalk#25] [Announcements]
# 2023/04/03/%E9%94%90%E8%AF%84/%E5%AE%B6%E6%97%8F%E8%AE%A1%E7%94%BB/

家族计划——破碎的人们所组成的圆前言田中罗密欧在《家族计划》这部作品中，想要传达的与《CROSS+CHANNEL》其实是同一个主题，不过前者更加注重于那份家庭之间的关系。这两部作品都从乖戾且难以理解，在世上孤身一人的角色入手，探讨人与人之间关系。在家计的故事中，更是多次直截了当地提出了那个命题——“人是不可能独自一人活下去的”。 尽管比起黑须太一那一群人，高屋敷一家人的黑暗扭曲属实是相形见绌了，但

https://dangokanako.github.io/2023/04/03/%E9%94%90%E8%AF%84/%E5%AE%B6%E6%97%8F%E8%AE%A1%E7%94%BB/

**Comment by MontagneWang** [comment]
在网上搜索「家族计画」时发现了博主的文章，很喜欢博主的文字      
家计作为本人最喜欢的中篇作品，很大程度上改变了我的生活观念      
作为坚定的独身主义者，曾自认一个人活着就好，无需与他人有太多深入的交流     
正如青叶所说：「世间到处都是欺瞒、伪善、谋杀、毒害、虚伪的誓言以及背叛。」      
但如今也明白这并非世界的全部，「硕果仅存的纯粹之地，只有毫无污秽地隐藏在人性之中的我们的爱而已。」    
人们所渴求的，就是某种能切实感受到与他人联系的事物，而这种事物，一定是由外界对你的在意程度所决定的     
---
也拜读了一下站主的其他博文，Muv-Luv 与 装甲恶鬼村正 也是我最喜欢的两部长篇作品🥰
